client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html

```diff
@@ -1,207 +1,198 @@
+                    </span>
+                    <span data-ng-if="!ctrl.remainingAvailableEmails"
+                        data-i18n-static="email_tab_modal_migrate_no_remaining_available_emails">
+                    </span>
+                    <span data-i18n-static="email_tab_modal_migrate_email_deleted"></span>
+                    <span data-i18n-static="email_tab_modal_migrate_email_reconfiguration"></span>
+                </p>
+                <!-- /After -->
+
+
+                <!-- Aknowledge Checkbox -->
+                <oui-field>
+                    <oui-checkbox name="aknowledge" required
+                        data-text="{{::tr('email_tab_modal_migrate_aknowledge')}}"
+                        data-model="ctrl.migrate.aknowledge">
+                    </oui-checkbox>
+                </oui-field>
+                <!-- /Aknowledge Checkbox -->
 
-                <button type="submit" class="oui-button oui-button_primary" data-i18n-static="email_tab_modal_migrate_submit" data-ng-disabled="!ctrl.migrate.password || !ctrl.migrate.aknowledge || ctrl.loaders.isWaitingForMigration"></button>
-                <button class="oui-button oui-button_link" data-i18n-static="common_cancel" data-ng-click="ctrlEmailDomainEmail.displayEmailsList()"></button>
             </div>
-        </form>
+            <!-- /Has available email address -->
 
-        <div class="text-center" data-ng-if="ctrl.loaders.isInitialRetrievalRunning || ctrl.loaders.isWaitingForDestinationEmails || ctrl.loaders.isWaitingForMigrationCheck || ctrl.loaders.isWaitingForMigration">
-            <oui-spinner></oui-spinner>
-        </div>
-        
-        <div data-ovh-alert="{{alerts.migrate}}"
-            data-ovh-alert-hide-remove-button>
-        </div>
+            <!-- No available email address -->
+            <div data-ng-if="!ctrl.migrate.destinationEmail">
+                <p>
+                    <span data-ng-bind-html="tr('email_tab_modal_migrate_no_available_email', ctrl.migrate.destinationService.name)"></span>
+                    <span data-i18n-static="email_tab_modal_migrate_configure_me"></span>
+                </p>
+                <p>
+                    <span data-i18n-static="email_tab_modal_migrate_choose_another_service_or"></span>
+                    <a class="oui-link oui-link_icon" href="/#/configuration/email_pro/{{::ctrl.migrate.destinationService.name}}?tab=ACCOUNT" target="_blank"
+                        data-ng-if="ctrl.migrate.destinationService.type === 'EMAIL PRO'">
+                        <span data-i18n-static="email_tab_modal_migrate_order_new_emails_button"></span>
+                        <span class="oui-icon oui-icon-external_link" aria-hidden="true"></span>
+                    </a>
+                    <a class="oui-link oui-link_icon" href="/#/configuration/exchange_hosted/{{::ctrl.migrate.destinationService.name}}/{{::ctrl.migrate.destinationService.name}}?tab=ACCOUNT" target="_blank"
```

Comment:
Does `target="_blank"` is required here?

Reply:
Removed too